docs/src/less/index.less

```diff
@@ -409,6 +409,7 @@ span.demo__inner {
 
 .demo-icon-boxes li {
     align-items: center;
+    background-color: var(--color-neutral-1);
```

Comment:
This doesn't work for dark mode, you should use `--color-background-secondary` instead

Reply:
Great point. Good catch.